tests/e2e/entities/test_CW20_balance_change.py

```diff
@@ -179,6 +196,19 @@ def filtered_cw20_balance_change_query(_filter):
                         self.assertIn(int(result["balanceOffset"]), entry["balance_offset"], "\nGQLError: fund amount does not match")
                         self.assertEqual(result["contract"], entry["contract"], "\nGQLError: contract address does not match")
 
+        with self.subTest("order by block height"):
+            for query, orderAssert in {
+                order_by_block_height_asc: self.assertGreaterEqual,
+                order_by_block_height_desc: self.assertLessEqual
+            }.items():
```

Comment:
If either of these sub-tests fails, it will likely require additional effort to figure out which one. Calling `self.subTest` inside the loop with unique names for each scenario should disambiguate their failure output. I think a different data structure might need to be in order to facilitate that but I also don't think we need anything fancy; it's python, after all. Something like this would likely suffice.

```suggestion
        for (name, query, orderAssert) in (
            ("order by block height ascending", order_by_block_height_asc, self.assertGreaterEqual),                        
            ("order by block height ascending", order_by_block_height_desc, self.assertLessEqual),
        ):        
            with self.subTest("order by block height"):            

```

Reply:
That's just the structure I was trying to think up - thanks!